spec/features/recipe_ingredients/delete_recipe_ingredient_page_spec.rb

```diff
@@ -1,17 +0,0 @@
-feature 'user deletes recipe ingredient' do
```

Comment:
Why removed?

Reply:
Can't remember now but I got this tests covered in previous pull request